xml/operations-compute-using_flavor_metadata.xml

```diff
@@ -8,154 +8,42 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_vhs_12v_vw">
  <title>Using Flavor Metadata to Specify CPU Model</title>
  <para>
-  In &kw-hos; Nova, the <literal>ComputeCapabilitiesFilter</literal>
-  scheduling filter matches the exact CPU model of the compute host and the
-  requested compute instance model. It doesn't entertain the libvirt's
-  capability of emulating the hosts CPU model to its guests if the guest's CPU
-  model is older or equal to that of the hosts.
+  In &kw-hos; &o_comp;, the ComputeCapabilitiesFilter checks the exact CPU
+  model of the compute host against the requested compute instance model. It
+  will only pick compute hosts that have the <literal>cpu_model</literal>
+  requested by the instance model, and if the compute host does not have that
+  <literal>cpu_model</literal>, the ComputeCapabilitiesFilter moves on to find
+  another compute host that matches, if possible. Selecting an outdated vCPU
+  model may cause &o_comp; to fail with <literal>no valid host found</literal>.
  </para>
  <para>
-  To assist, we have introduced a new Nova scheduler filter which captures the
-  cpu_models as a subset of a particular CPU family. The filter determines if
-  the host CPU model is capable of emulating the guest CPU model by
-  maintaining the mapping of the vCPU models per vendor (currently Intel and
-  AMD) and compares it against the 'model' of the host.
+  To assist, there is a &o_comp; scheduler filter that captures <literal>
+  cpu_models</literal> as a subset of a particular CPU family. The filter
```

Comment:
Don't put a line break directly after the opening tag `<literal>`: That will introduce an ugly leading space which you will see because literals are underlined in our layout.